mission/inspect-train-serve-impl.md

```diff
@@ -1,10 +1,10 @@
-# Inspect Training and Serving implementation
+# Inspect Training and Serving Implementation
 
 First the model needs to be trained using the inspections training
 data. Then one can serve the model to predict the _overall inspection score_ and the _lowest
 partial score_ based on new data about an inspection.
 
-To take a look at the training in full detail you can find the code [here](\*\*\*link missing). The code for
+To take a look at the training in full detail you can find the code [here](../src/train/). The code for
```

Comment:
Looks like in line 8, for the serving code, it was pointing to `../src/train/train.py` all this time as opposed to now `../src/serve/`—can you change this as well?

Otherwise, looks good!

Reply:
I changed it